packages/grid/stories/grid-cellValidation.stories.tsx

```diff
@@ -0,0 +1,334 @@
+import {
+  CellEditor,
+  DropdownCellEditor,
+  Grid,
+  GridCellValueProps,
+  GridColumn,
+  NumericCellEditor,
+  NumericColumn,
+  TextCellEditor,
+} from "../src";
+import { Story } from "@storybook/react";
+import { faker } from "@faker-js/faker";
+import { useCallback, useState } from "react";
+import "./grid.stories.css";
+import { StatusIndicator } from "@salt-ds/core";
+
+export default {
+  title: "Data Grid/Data Grid",
+  component: GridColumn,
+  argTypes: {},
+};
+
+const discountOptions = ["-", "5%", "10%", "20%", "50%"];
+const discountValues = [1, 0.95, 0.9, 0.8, 0.5];
+
+interface RowExample {
+  id: string;
+  name: string;
+  description: string;
+  amount: number;
+  price: number;
+  discount: string;
+  status?: "error" | "warning" | "success";
```

Comment:
i'm wondering if it would be good to export the CellValidationStatus type?

Reply:
Yeah, probably